src/test/mocks/StrategyManagerMock.sol

```diff
@@ -84,6 +84,8 @@ contract StrategyManagerMock is
         thirdPartyTransfersForbidden[strategy] = value;
     }
 
+    function strategyIsWhitelistedForDeposit(IStrategy strategy) external view returns (bool) {}
```

Comment:
Does this defer to false?

Reply:
Can ignore this for now, will add testing once we have enough agreement on implementation.